web/src/components/molecules/WorkspaceSettings/index.tsx

```diff
@@ -0,0 +1,33 @@
+import ContentSection from "@reearth-cms/components/atoms/ContentSection";
+import InnerContent from "@reearth-cms/components/atoms/InnerContents/basic";
+import DangerZone from "@reearth-cms/components/molecules/WorkspaceSettings/DangerZone";
+import WorkspaceForm from "@reearth-cms/components/molecules/WorkspaceSettings/Form";
+import { useT } from "@reearth-cms/i18n";
+
+export type Props = {
+  workspaceName?: string;
+  onWorkspaceUpdate: (name?: string | undefined) => Promise<void>;
+  onWorkspaceDelete: () => Promise<void>;
+};
+
+const WorkspaceSettings: React.FC<Props> = ({
+  workspaceName,
+  onWorkspaceUpdate,
+  onWorkspaceDelete,
+}) => {
+  const t = useT();
+
+  return (
+    <InnerContent title="Workspace setting">
+      <ContentSection title={t("General")}>
+        <WorkspaceForm onWorkspaceUpdate={onWorkspaceUpdate} workspaceName={workspaceName} />
```

Comment:
Please start to pay attention to prop order and keep it the same as the Props type of the component you are passing to.
Like I mentioned in another PR's comment, try to keep variables first, followed by functions.

Reply:
@KaWaite , changed, and I'll keep this in mind for future.